eng/common/scripts/Verify-Links.ps1

```diff
@@ -193,7 +193,15 @@ function CheckLink ([System.Uri]$linkUri)
         }
         else {
           Write-Host "Exception while requesting $linkUri"
-          Write-Host $_.Exception.ToString()
+          if ($linkUri.ToString().StartsWith("#")) {
```

Comment:
This doesn't seem like the correct place for this check. I'd suggest it go under the checkLinkGuidance block below.

Reply:
We can catch the link here. 

https://github.com/Azure/azure-sdk-for-c/pull/1387/checks?check_run_id=1190491733
There is another purpose to put it here.
We log out the error messages for anchor link which is not necessary. So I suppress the exception messages for anchor link case.
![image](https://user-images.githubusercontent.com/48036328/94752595-8a6b2500-0340-11eb-88c8-1250bb51d56d.png)

Reply:
Do we want to update the link guidance of the anchor link? If so, we will put it under the checkLinkGuidance section.

Reply:
Once we figure out the correct guidance for anchor links then we should definitely update our guidelines and add it under the link guidance enforcement section.